### ISSUE TITLE
gcp fleet collection feedback

### ISSUE THREAD

**Issue by connorjclark** [GoogleChrome/lighthouse#11267]
1. `--kill` prompts Y/N for deleting, which slows down the process when there are tons of instances

![image](https://user-images.githubusercontent.com/4071474/90194104-15da2800-dd8c-11ea-8a99-5687bebbd3b7.png)

2. the output was going to fast when I created it, but I noticed some instances were not created. I think the error was "CPU limit" or something? maybe "compute limit".

3. I don't really care about the trace data, is there a way to make that optional to collect or download?

**Comment by patrickhulce** [comment]
Go for any and all of it :) I personally like the confirmation, but can see the need for auto. Does your GCP account have a limit on instance count? `--skip-trace-download`? It lets you rerun the exact flow with `lighthouse -A` which I think is pretty handy for debugging and reproducibility

**Comment by Beytoven** [comment]
So I've been using the scripts a lot over the last couple of days and have already started to make a couple changes. I'll put up a PR soon that includes new scripts for extracting and doing some quick analysis on the data as well.

**Comment by Beytoven** [comment]
A couple of things I want to add here. Something that would be nice is being able to configure the number of URLs  run on a machine when running the fleet script. Running the fleet as is with a large data set could be pretty expensive. Another slightly related thing I'd like is for the script to be able to take in a config file that can handle the above config as well as other things such as the branch to use or any of the other env variables that can be set prior to running the script.

**Comment by Beytoven** [comment]
<img width="831" alt="Screen Shot 2020-09-03 at 10 49 45 AM" src="https://user-images.githubusercontent.com/27858957/92149831-649b4080-edd3-11ea-80c6-7fd6992d45f7.png">
Did a run last night and all but one of the instances ran to completion. Got the log from that instances and it looks like it got hung in the middle of a LH run. This a bit different than the other times I've seen failures which is usually due to `collect.log` not being found.

**Comment by patrickhulce** [comment]
I ran into this a few times with the massive 3P collection as well. I ended up with a cron that killed Chromium PIDs if they had been alive for too long as that ends up triggering the retry logic. Maybe we port something similar over to here?

**Comment by Beytoven** [comment]
Yeah, that could work. I was thinking of implementing some sort of timeout although I'm not very proficient in BASH so I can't say how feasible/difficult that would be.
